content/app/development/configuration/email/_index.en.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Email
+description: How to create custom email notifications for apps.
+toc: true
```

Comment:
Consider if ToC makes sense for such a small page

Reply:
It does not. Removed.